client/src/app/main-menu/settings.tsx

```diff
@@ -182,6 +183,9 @@ function Settings({setMenuSection}: Props) {
 						</div>
 					</div>
 				</div>
+				<Button variant="stone" onClick={handleHallOfFame}>
```

Comment:
This should probably be moved to within the stats box or at the bottom of the settings grid.

Reply:
Yep, this looks better.